central/detection/lifecycle/manager_impl.go

```diff
@@ -248,6 +253,19 @@ func (m *managerImpl) buildMapAndCheckBaseline(indicatorSlice []*storage.Process
 	}
 }
 
+func (m *managerImpl) SendBaselineToSensor(pw *storage.ProcessBaseline) {
+	err := m.connectionManager.SendMessage(pw.GetKey().GetClusterId(), &central.MsgToSensor{
+		Msg: &central.MsgToSensor_BaselineSync{
+			BaselineSync: &central.BaselineSync{
+				Baselines: []*storage.ProcessBaseline{pw},
+			}},
+	})
+	if err != nil {
+		log.Errorf("Error sending process baseline to cluster %q: %v", pw.GetKey().GetClusterId(), err)
+	}
+	log.Infof("Successfully sent process baseline to cluster %q: %s", pw.GetKey().GetClusterId(), pw.GetId())
```

Comment:
Need a return in the `err != nil` case.  Otherwise we will log an error and then success.